src/app/shared/module/material/service/snack-bar.service.ts

```diff
@@ -33,15 +33,17 @@ export class SnackBarService {
     return from(
       forkJoin([this.translate.get(message), action ? this.translate.get(action) : of(undefined)])
         .pipe(
-          flatMap(([translatedMessage, translatedAction]) =>
-            this.matSnackBar
-              .open(translatedMessage, translatedAction, {
-                duration: 5 * 1000,
-                verticalPosition: 'bottom',
-                panelClass: ['snack-bar-service', panelClass],
-              })
-              .onAction()
-          )
+          flatMap(([translatedMessage, translatedAction]) => {
+            let snackBar = this.matSnackBar.open(translatedMessage, translatedAction, {
+              duration: 5 * 10000,
+              verticalPosition: 'bottom',
+              panelClass: ['snack-bar-service', panelClass],
+            })
+            let snackBarElement = document.getElementsByClassName('snack-bar-service')[0]
```

Comment:
i recommend `document.querySelection('.snack-bar-service');` and then to further double check that the `snackBarElement` exists before trying to do operations with it

Reply:
Added more validation (basically changed it to a `querySelectionAll` and a `forEach` loop (done in a482ced).
Also fixed the duration value (I've increased it for debug purposes, but I forgot to revert that change)